src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -13512,14 +13528,14 @@ namespace OutputReportTabular {
                     compLoad.peakDateHrMin = CoolPeakDateHrMin(zoneIndex);
                 }
 
-                // Outside  Dry Bulb Temperature
+                // Outside Dry Bulb Temperature
                 compLoad.outsideDryBulb = CalcFinalZoneSizing(zoneIndex).CoolOutTempSeq(timeOfMax);
 
-                // Outside  Wet Bulb Temperature
+                // Outside Wet Bulb Temperature
                 // use standard sea level air pressure because air pressure is not tracked with sizing data
                 if (CalcFinalZoneSizing(zoneIndex).CoolOutHumRatSeq(timeOfMax) < 1.0 &&
                     CalcFinalZoneSizing(zoneIndex).CoolOutHumRatSeq(timeOfMax) > 0.0) {
-                    compLoad.outsideWebBulb = PsyTwbFnTdbWPb(CalcFinalZoneSizing(zoneIndex).CoolOutTempSeq(timeOfMax),
+                    compLoad.outsideWetBulb = PsyTwbFnTdbWPb(CalcFinalZoneSizing(zoneIndex).CoolOutTempSeq(timeOfMax),
```

Comment:
Funny how a typo just proliferates when you copy and paste it :-)
Thanks for fixing these.